### ISSUE TITLE
Write to video

### ISSUE THREAD

**Issue by AIMusicExperiment** [WASasquatch/was-node-suite-comfyui#162]
I'm really excited to use some of you nodes, but as with most developers you have offered very little in the way of documentation. When using the write to video node, using a batch from your batch loader. I get a resize error.  Can you enlighten me as to what I'm doing incorrectly? The same files work just fine with the create a video from a path node.  But for this workflow that isn't ideal.  I wish to generate an image and have it write to a video, so that when the batch is done I have a video. Can you help me out? Thanks!

**Comment by WASasquatch** [comment]
That's the goal of the node. But without the error, workflow with what you're actually doing, it's hard to say.

**Comment by pennyvc** [comment]
[https://drive.google.com/file/d/1iSCt9_YFzHtOzBOaRcvk0d8Dzg2mMuKy/view?usp=sharing](url)
![Capture](https://github.com/WASasquatch/was-node-suite-comfyui/assets/128430314/0a8fca4e-c36c-40f3-9eb2-a59f540a20ae)
happens when using H.264 with this setup.
Was having trouble getting ffmpeg installed but thankfully I realized you need to double the \\  in the path, suggest you append this to the video node section so people don't try just copy paste path.

**Comment by WASasquatch** [comment]
I can't reproduce this with my own images in your workflow. 

Either a image is broken, corrupted somehow, or maybe your opencv installation is bugged, or wrong version.

**Comment by pennyvc** [comment]
yeah, pretty sure issue is incorrect opencv

**Comment by WASasquatch** [comment]
Maybe try `pip install --force-reinstsll --update opencv-python-headless[ffmpeg]` on your ComfyUI python environment.

**Comment by pennyvc** [comment]
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
basicsr 1.4.2 requires future, which is not installed.
basicsr 1.4.2 requires tb-nightly, which is not installed.
facexlib 0.3.0 requires filterpy, which is not installed.
tensorflow-intel 2.13.0 requires astunparse>=1.6.0, which is not installed.
tensorflow-intel 2.13.0 requires gast<=0.4.0,>=0.2.1, which is not installed.
tensorflow-intel 2.13.0 requires google-pasta>=0.1.1, which is not installed.
tensorflow-intel 2.13.0 requires grpcio<2.0,>=1.24.3, which is not installed.
tensorflow-intel 2.13.0 requires h5py>=2.9.0, which is not installed.
tensorflow-intel 2.13.0 requires tensorboard<2.14,>=2.13, which is not installed.
numba 0.57.1 requires numpy<1.25,>=1.21, but you have numpy 1.25.2 which is incompatible.
tensorflow-intel 2.13.0 requires keras<2.14,>=2.13.1, but you have keras 2.10.0 which is incompatible.
tensorflow-intel 2.13.0 requires numpy<=1.24.3,>=1.22, but you have numpy 1.25.2 which is incompatible.
tensorflow-intel 2.13.0 requires tensorflow-estimator<2.14,>=2.13.0, but you have tensorflow-estimator 2.10.0 which is incompatible.
tensorflow-intel 2.13.0 requires typing-extensions<4.6.0,>=3.6.6, but you have typing-extensions 4.7.1 which is incompatible.

:(  eeep

**Comment by pennyvc** [comment]
annnd yep, broke comfyui

**Comment by WASasquatch** [comment]
Did you add the paths it tells you to your PATH environment variable? 

According to this post their may be a breaking change in  4.8.0.76 too and .74 may be fix: https://github.com/WASasquatch/was-node-suite-comfyui/issues/168#issuecomment-1686672133

**Comment by pennyvc** [comment]
yeah, fixed path issues, still get torch depreciation though, no idea what thats from, looks like some kind of bug in numpy meta process or something like that.
also eventually fixed yeh, had to reinstall with different version.
